tests/test_rules_mapper_holdings.py

```diff
@@ -29,3 +32,44 @@ def test_get_marc_textual_stmt_correct_order_and_not_deduped():
         all_stmts = [f["statement"] for f in rec["holdingsStatements"]]
         all_94s = [f for f in all_stmts if stmt == f]
         assert len(all_94s) == 2
+
+
+def test_remove_from_id_map():
+    mocked_rules_mapper_holdings = Mock(spec=RulesMapperHoldings)
+    mocked_rules_mapper_holdings.holdings_id_map = {
+        "h15066915": "5a0af31f-aa4a-5215-8a60-712b38cd6cb6",
+        "h14554914": "c9c44650-11e2-5534-ae50-01a1aa0fbd66",
+    }
+
+    former_ids = ["h15066915"]
+
+    RulesMapperHoldings.remove_from_id_map(mocked_rules_mapper_holdings, former_ids)
+
+    # The ids in the former_ids have been removed, any others are still there
+    assert "h15066915" not in mocked_rules_mapper_holdings.holdings_id_map.keys()
+    assert "h14554914" in mocked_rules_mapper_holdings.holdings_id_map.keys()
```

Comment:
I am pretty sure you can ommit the .keys() here, but it certainly makes it clearer.

Reply:
That's how I had it from the start -- but my impression was that that was making the tests pass even when they shouldn't and that adding the .keys() fixed this.